exercises/book-store/canonical-data.json

```diff
@@ -0,0 +1,86 @@
+{
+   "book-store": {
+      "description": "Calculate lowest price for shopping basket only containing books from a single series",
+      "cases": [
+         {
+            "description": "A basket containing only a single book is charged full price.",
+            "basket": [1],
+            "expected": 8.0
+         },
+         {
+	    "description": "A basket containing only two of the same book is charged full price.",
+            "basket": [2, 2],
+            "expected": 16.0
+         },
+         {
+            "description": "No charge to carry around an empty basket.",
+            "basket": [],
+            "expected": 0.0
+         },
+         {
+	    "description": [
+		             "A basket containing two different books is the cheapest grouping, ",
+		             "better than purchasing each book on its own."
+		           ],
+            "basket": [1, 2],
+            "expected": 15.2
+         },
+         {
+	    "description": "A basket of three different books is cheaper than grouping the books any other way.",
+            "basket": [1, 2, 3],
+            "expected": 21.60
+         },
+         {
+	    "description": "A basket of four different books is cheaper than grouping the books any other way.",
+            "basket": [1, 2, 3, 4],
+            "expected": 25.60
+         },
+         {
+	    "description": "A basket of five different books is cheaper than grouping the books any other way.",
+            "basket": [1, 2, 3, 4, 5],
+            "expected": 30.00
+         },
+         {
+	    "description": [
+		             "A basket containing three pairs of three different books plus ",
+		             "one copy each of the other two books, not paired, is the least expensive ",
```

Comment:
I don't think the "other two books, not paired" in this description is accurate. The other two books *do* get grouped - the 4 gets grouped with one [1, 2, 3] set, and the 5 gets grouped with the other [1, 2, 3] group. The wording seems to imply that the groups in this basket would be [1, 2, 3], [1, 2, 3], [4], [5], when in fact it is [1, 2, 3, 4], [1, 2, 3, 5]. I would like to see it made clear that [1, 2, 3, 4], [1, 2, 3, 5] is the best grouping, rather than [1, 2, 3, 4, 5], [1, 2, 3] (or any other).